app/policies/booking_policy.rb

```diff
@@ -21,6 +21,7 @@ def update?
   end
 
   def destroy?
+    false
     # only admin can destroy and have the status of the booking go to completed or cancelled? Similar to Amazon,order history can't be deleted
```

Comment:
ah, so currently destroy is simply false? Yeah maybe go for admin then I guess?

Reply:
Can we admin for later for now and focus on the core journey with the users. It shouldn't cause any problems since we just won't have any links or buttons with a delete booking method